Kickstarter-iOS/AppDelegateViewModel.swift

```diff
@@ -965,6 +983,27 @@ public final class AppDelegateViewModel: AppDelegateViewModelType, AppDelegateVi
   public let updateConfigInEnvironment: Signal<Config, Never>
 }
 
+private func atTrackingAuthorizationStatus() -> ATTrackingAuthorizationStatus {
+  var authorizationStatus: ATTrackingAuthorizationStatus = .notDetermined
+
+  DispatchQueue.main.asyncAfter(deadline: .now() + 1) {
```

Comment:
The app has to be completely active before the dialog can be shown. Since AppDelegate doesn't have a `viewDidAppear` method, this seems to be the standard way to approach this on app launch.

Reply:
yep - I replied and will address your other comment. I think we are commenting at the same time lol